bucoffea/limit/limit.py

```diff
@@ -32,7 +32,7 @@ def main():
     else:
         acc = acc_from_dir(args.inpath)
 
-    outdir = pjoin('./output/',os.path.basename(args.inpath))
+    outdir = pjoin('./output/',list(filter(lambda x:x,args.inpath.split('/')))[-1])
```

Comment:
I would vote for `os.path.basename(os.path.dirname(args.inpath))`

Reply:
I think Andreas is right, `os.path.basename()` just returns an empty string if `args.inpath` ends with a slash, the updated `outdir` seems to work well with both cases though.

Reply:
I think `os.path.basename(os.path.dirname(args.inpath))` doesn't catch the right directory name when `args.inpath` doesn't end with a slash, but the updated `outdir` seems to work well for both cases.

Reply:
Basically `os.path.dirname` just removes the trailing slash, and then `os.path.basename` can catch it.

Reply:
Yeah you're right that it works when `args.inpath` ends with a slash. But in the case where `args.inpath` doesn't end with a slash, say `inpath='ab/cd'`, `os.path.dirname` returns `ab` only, so `os.path.basename` command catches `ab` instead of `cd`.

Reply:
emmm you are right.

Reply:
thanks for checking this out! Indeed, this was the only option I could come up with that covers both cases.